rubin_sim/scheduler/utils/sky_area.py

```diff
@@ -9,77 +9,90 @@
-    the galactic plane (bulge and background), the Northern Ecliptic Spur, and
-    the Southern Celestial Pole. The individual regions are stored in a dictionary of
-    self.maps (and self.maps_perfilter) that are simple masks (0-1) which when multiplied
-    by their relevant self.nvis dictionary items, produce the expected contribution toward the
-    final survey footprint. The total survey footprint, when combined across regions, is
-    stored in self.total (and self.total_perfilter), which is a footprint weighted by the
-    expected number of visits per pointing; i.e. the value of 'total in the dust-free WFD region
-    reflects the expected number of visits per pointing in the dust-free WFD.
-    Note that this final 'total' map is not limited by the actual amount of survey time -- you can
-    use tools in lsst-pst/survey_strategy/survey_utils to estimate how much time would be required
-    to complete a particular footprint. The scheduler simply uses a weighted map to prioritize
-    observations, so the achieved footprint will reflect the ratio of these inputs (combined
-    with the amount of time available in any part of the sky).
+    """
+    Generate survey footprint maps in each filter.
 
     Parameters
     ----------
-    nside : `int`, opt
-        Healpix resolution for map. Default 64.
-    nvis_wfd_default : `int`, opt
-        Goal number of visits in the WFD region.
-        This acts as a scaling factor for the maps in each region, but in practice the
-        scheduler will achieve as many visits in each region as possible (keeping the overall balance).
-        Combined with a function to calculate the survey fraction required for a given number of visits
-        over a given area, this scaling factor can provide an estimate of the 'likely achieved' number of
-        visits from a simulation. Default 860.
-    nvis_frac_nes : `float`, opt
-        Fraction of WFD-level visits to spend in the NES pointings.
-        Default 0.3  (0.3 * 860 WFD visits = ~258 visits per pointing in the NES).
-    nvist_frac_gp : `float`, opt
-        Fraction of WFD-level visits to spend in the background (non-bulge/WFD) galactic plane regions.
-        Default 0.27
-    nvis_frac_scp : `float`, opt
-        Fraction of WFD-level visits to spend in the south celestial pole region (which acts as a
-        'backup' sky coverage all the way up to ~dec_max, beyond the SCP, in case of 'holes' in sky coverage).
-        Default 0.14
-    dec_max : `float`, opt
-        Maximum declination value for sky coverage before it is reset by defining the dust-free WFD regions.
-        This is useful if other regions are defined before the dust-free WFD, but otherwise is overriden.
-        In degrees, default 12.
+    nside : `int` (32)
+        Healpix nside (32)
+    dust_limit : `float` (0.199)
+        E(B-V) limit for dust extinction. Default of 0.199.
+    smoothing_cutoff : `float` (0.45)
+        Magic number (E(B-V)?)
```

Comment:
I suggest combining this parameter's explanation and the one below (smoothing_beam).